sentry/src/db.rs

```diff
@@ -2,12 +2,20 @@ use redis::aio::MultiplexedConnection;
 use redis::RedisError;
 
 use bb8::Pool;
-use bb8_postgres::{
-    tokio_postgres::{Error as PostgresError, NoTls},
-    PostgresConnectionManager,
-};
+use bb8_postgres::tokio_postgres::NoTls;
+use bb8_postgres::PostgresConnectionManager;
 use lazy_static::lazy_static;
 
+pub type DbPool = Pool<PostgresConnectionManager<NoTls>>;
+
+pub mod field {
```

Comment:
In order not to add the whole `tokio_postgres` in primitives. We might add it as a feature, but don't think we should do it as it specifically for sentry and it's persistent mechanism and DB.

Reply:
this sounds okay